src/sentry/utils/data_scrubber.py

```diff
@@ -46,17 +46,24 @@ class SensitiveDataFilter(object):
     Asterisk out things that look like passwords, credit card numbers,
     and API keys in frames, http, and basic extra data.
     """
-    # http://www.richardsramblings.com/regex/credit-card-numbers/
-    VALUES_RE = re.compile(r'\b(?:3[47]\d|(?:4\d|5[1-5]|65)\d{2}|6011)\d{12}\b')
+    VALUES_RE = re.compile(r'|'.join([
+        # http://www.richardsramblings.com/regex/credit-card-numbers/
+        r'\b(?:3[47]\d|(?:4\d|5[1-5]|65)\d{2}|6011)\d{12}\b',
+        # various private/public keys
+        r'-----BEGIN[A-Z ]+(PRIVATE|PUBLIC) KEY-----.+-----END[A-Z ]+(PRIVATE|PUBLIC) KEY-----',
+        # social security numbers (US)
+        r'^\b\d{3}-?\d{2}-?\d{4}\b',
```

Comment:
Agreed. The issue we had before with credit card numbers was that basically every timestamp value was being filtered out because it was a just simply a 16 digit number. The same situation would apply here for simply any 9 digit number.